OpenUpSA/wazimap-ng
# Added new field color in profile category #181

## Description
Add new field named color in profile category model.
Use django-colorfield for creating modal field and showing admin widget
Add color field in all_details api

## Related Issue
#181 

## How to test it locally
Go to profile category and click on text field for color.
Selected color Hex value should be visible in text box. 
If left empty api will have empty string as value.

<img width="991" alt="Screenshot 2021-01-08 at 3 33 06 PM" src="https://user-images.githubusercontent.com/6871866/104002470-240b6380-51c7-11eb-9f2b-586b0ddb909e.png">
<img width="1019" alt="Screenshot 2021-01-08 at 3 33 18 PM" src="https://user-images.githubusercontent.com/6871866/104002479-279eea80-51c7-11eb-965c-a0f4c2872712.png">


## Changelog
* Added new package : django-colorfield
* Added new field in models.py and migration
* Changed admin.py for profile collection to show color widget
* Added color field in all details api
* Added colorfield in config apps


### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
